packages/@webex/plugin-meetings/test/unit/spec/meetings/index.js

```diff
@@ -593,11 +593,24 @@ describe('plugin-meetings', () => {
               );
               MeetingUtil.cleanUp = sinon.stub().returns(Promise.resolve());
             });
-            it('destroy only non active locus meetings and keep active locus meetings and any other non-locus meeting', async () => {
+
+            it('destroy any meeting that has no active locus url if keepOnlyLocusMeetings', async () => {
```

Comment:
nitpick: For completeness, a version of this test where `true` is explicitly passed in would be nice, but not essential!